### PR TITLE
LibGfx: Use AntiAliasingPainter::fill_path() for drawing font glyphs

### PR DESCRIPTION
**Pull request by MacDue**
Using the general AA painter fill_path() is indistinguishable from the previous rasterizer, so this switch simply allows us to share more code.

Screenshot from 2023-07-09 20-40-25_skip_no pdf

### CODE REVIEW

File: Userland/Libraries/LibGfx/Font/OpenType/Font.h [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 #pragma once
 
+#include <AK/HashMap.h>
```
Comment (Hendiadyoin1) [inline Userland/Libraries/LibGfx/Font/OpenType/Font.h:9]: This include change looks unrelated or am I missing a transient include change here?

Comment (MacDue) [reply Userland/Libraries/LibGfx/Font/OpenType/Font.h:9]: This was a transitive include (which the commit does not compile  without)

Comment (MacDue) [reply Userland/Libraries/LibGfx/Font/OpenType/Font.h]: Looks like it can be removed (it did appear to be needed at one point in this refactor). Though nothing is explicitly including HashMap (so it's getting it from some random include or includer)
